.gitignore

```diff
@@ -1,5 +1,5 @@
 # dependencies
-node_modules
+**node_modules**
```

Comment:
What's the benefit of this?

Reply:
It's not picking up node_modules inside my packages 😞

Reply:
I think `node_modules/` should work for all `node_modules` directories, couldn't find any info on `**node_modules**` so wanted to confirm.

Reply:
Okay I take the blame. My editor screwed up and was not respecting the `.gitignore` properly. I've reverted this. Thanks for pointing out

Reply:
I once wasted a couple of hours trying to figure out why a filename change wasn't being tracked by git (because of case insensitive nature of macOS) 😄